labellab-client/src/actions/auth.js

```diff
@@ -235,11 +238,12 @@ export const GithubOauth = (credentials, callback) => {
       headers: {
         'content-type': 'application/json',
         'Access-Control-Allow-Origin': 'http://localhost:3000',
-        'Accept': 'application/json'
```

Comment:
please include `accept` in inverted commas

Reply:
My code formatter must have changed them. I'll change it back.